client/state/plugins/installed/actions.js

```diff
@@ -447,7 +575,7 @@ export function fetchPlugins( siteIds ) {
 
 				data.plugins.map( ( plugin ) => {
 					if ( plugin.update && plugin.autoupdate ) {
-						updatePlugin( siteId, plugin )( dispatch );
+						updatePlugin( siteId, plugin )( dispatch, getState );
```

Comment:
nit: out of curiosity, would `dispatch( updatePlugin( siteId, plugin ) )` work here as well?

Reply:
Yes 👍  

AFAIK `dispatch( actionCreator() )` is the more recommended way to dispatch Redux actions, but since this was done like this in multiple locations since the previous plugins reduxification effort, I'd like to find all of these and replace them separately in another PR. Making a note and will follow-up.

Thanks for the review 🙌